File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.java
class Solution {
    public int canBeTypedWords(String text, String bro) {
        boolean []arr=new boolean[26];
        for(char c: bro.toCharArray()){
            arr[c-'a']=true;
        }
        int count=0;
        boolean flag=true;
        for(int i=0;i<text.length();i++){
            if(text.charAt(i)!=' '){
                if(arr[text.charAt(i)-'a']){
                    flag=false;
                }
            }
            else{
                if(flag)count++;
                flag=true;
            }
        }
        if(flag)count++;
        return count;
    }
}

File: 1264-maximum-number-of-words-you-can-type/maximum-number-of-words-you-can-type.cpp
class Solution {
public:
    int canBeTypedWords(string text, string bro) {
       stringstream ss(text);
       string ans;
       int count=0;
       while(getline(ss,ans,' ')){
         bool flag=true;
          for(int i=0;i<bro.size();i++){
            for(int j=0;j<ans.size();j++){
                if(bro[i]==ans[j]){
                    flag=false;
                    break;
                }
            }
            if(flag==false)break;
          }
            if(flag==true){
                count++;
            }
          
       }
       return count;
    }
};